app/src/main/res/layout/statistics_listitem_total.xml

```diff
@@ -1,44 +1,48 @@
 <?xml version="1.0" encoding="utf-8"?>
 <RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
-                xmlns:tools="http://schemas.android.com/tools"
-                android:layout_width="match_parent"
-                android:layout_height="wrap_content"
-                android:padding="16dp">
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:padding="16dp">
 
     <de.danoeh.antennapod.view.PieChartView
-            android:id="@+id/pie_chart"
-            android:layout_width="wrap_content"
-            android:layout_height="wrap_content"
-            android:layout_centerInParent="true"
-            android:layout_marginLeft="8dp"
-            android:layout_marginRight="8dp"
-            android:maxWidth="800dp"
-            android:minWidth="460dp" />
+        android:id="@+id/pie_chart"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_centerInParent="true"
+        android:layout_marginRight="8dp"
+        android:maxWidth="800dp"
+        android:minWidth="460dp"
+        android:layout_marginEnd="8dp" />
```

Comment:
Isn't this asymmetric, now? It now only has a right margin.

Reply:
sorry about that